packages/edit-site/src/components/style-book/index.js

```diff
@@ -249,16 +249,18 @@ function StyleBook( {
 				{ showTabs ? (
 					<div className="edit-site-style-book__tabs">
 						<Tabs>
-							<Tabs.TabList>
-								{ tabs.map( ( tab ) => (
-									<Tabs.Tab
-										tabId={ tab.name }
-										key={ tab.name }
-									>
-										{ tab.title }
-									</Tabs.Tab>
-								) ) }
-							</Tabs.TabList>
+							<div className="edit-site-style-book__tablist-container">
```

Comment:
Do we need this extra `div`, or could we assign styles to the tablist directly?

Reply:
Feel free to try, and I don't remember the details now but I do remember arriving to the conclusion that it was necessary. Trust me, I always try to go the minimal route :) - I could've missed something though!